tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/AwsLambda/LambdaTestToolFixture.cs

```diff
@@ -40,6 +40,7 @@ public LambdaTestToolFixture(RemoteApplication remoteApplication, string newReli
 
                     // Finest level logs are necessary to read the uncompressed payloads from the agent logs
                     remoteApplication.NewRelicConfig.SetLogLevel("finest");
+                    remoteApplication.NewRelicConfig.SetDebugStartupDelaySeconds(20); // TODO: Testing only
```

Comment:
I think you can drop this now?

Reply:
Good catch. That's why I added the TODO!